src/routes/rules/RulesView.vue

```diff
@@ -313,72 +277,72 @@ export default {
     BaseVideo
   },
   setup() {
+    const { t } = useI18n();
     const { logoSrc } = useThemedLogo();
     return {
+      t,
       logoSrc,
     };
   },
   computed: {
     ...mapStores(useAuthStore),
     buttonText() {
       if (this.authStore.username) {
-        return 'Find a Game';
+        return `${this.$t('rules.findGame')}`;
```

Comment:
```suggestion
        return this.t('rules.findGame');
```

Reply:
Same here, you should be able to do it like this in all the translations inside the script tag

Reply:
Resolved in commit 0584c1c